python/models/model.py

```diff
@@ -0,0 +1,42 @@
+"""

```

Comment:
Hi @akashAD98 My I know what's the purpose of ```model.py``` and ```custom_segmentation.py```?

Reply:
okay thanks ill remove &unnesessory componets

Reply:
i deleted custom_segmentation.py & kept model.py bcz we are importing pre ,post code from model file,
without that im getting no module name found error